### DISCUSSION TITLE
Improve and unify error reporting across platforms

### DISCUSSION THREAD

**Discussion by armands-malejevs** [TheWidlarzGroup/react-native-video#4945] [Ideas]
### What problem are you trying to solve?

Currently our error reporting is limited and in some cases not even useful when providing a string from resources instead of the actual error message.

It is best if there is consistency across all platforms and errors have useful and correct information included in them instead of identical generic messages that not always represent the issue correctly.

### Proposed solution

We should unify error codes across all platforms and only use actual error messages.

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
